### PR TITLE
Update Docs for Latest SDK Release (0.9)

### PR DESCRIPTION
**Pull request by qryptdev**
Staging PR for SDK release. To-Do: 
- Validate entropy projection changes
- Update version to exact version released: 0.9.x
- Staging: https://webstage.z13.web.core.windows.net/docs/
   - Entropy changes: https://webstage.z13.web.core.windows.net/docs/concepts/entropy-projection/
   - Changelogs: https://webstage.z13.web.core.windows.net/docs/sdk/changelogs/

### CODE REVIEW

File: content/sdk/changelogs/_index.md [hunk 1]
```diff
@@ -6,10 +6,20 @@ draft: false
...
-4. [SDK v0.6.4]({{<ref "#sdk-v064" >}})
+1. [SDK v0.9.0]({{<ref "#sdk-v090" >}})
+2. [SDK v0.8.6]({{<ref "#sdk-v086" >}})
+3. [SDK v0.7.10]({{<ref "#sdk-v0710" >}})
+4. [SDK v0.7.4]({{<ref "#sdk-v074" >}})
+5. [SDK v0.6.4]({{<ref "#sdk-v064" >}})
+
+---
+
+### SDK v0.9.0
```
Comment (qryptdev) [inline content/sdk/changelogs/_index.md:17]: This will be v0.9.2

File: content/sdk/changelogs/_index.md [hunk 2]
```diff
@@ -6,10 +6,20 @@ draft: false
...
+**Notable Changes:**
+
+1. Change backend to use sequential sampling and seedless extraction
+2. Changed smallest allowed key size to 1 byte
```
Comment (qryptdev) [inline content/sdk/changelogs/_index.md:21]: Use present tense, "Change" and "Simplify"

Comment (qryptdev) [reply content/sdk/changelogs/_index.md]: Fixed!

**Comment by qryptdev** [comment]
Jay - Approved

**Review by alison-qrypt — APPROVED** [review]

**Review by wmcrobertsq — APPROVED** [review]

**Pull request merged**